docs/examples/extensions/add-report/js/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { addFilter } from '@wordpress/hooks';
+import { Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * WooCommerce dependencies
+ */
+import { ReportFilters } from '@woocommerce/components';
+
+const Report = ( { path, query } ) => {
+	return (
+		<Fragment>
+			<ReportFilters
+				query={ query }
+				path={ path }
+				filters={ [] }
+				advancedFilters={ {} }
+			/>
```

Comment:
I think it would be helpful to add a quick example table here; just a quick table to show fake data so the page isn't empty.

Reply:
I'd like to add some coherency to the fake data, like a theme of sorts. So, I'd like to tackle this separately as well.